.webpack/webpack.common.mjs

```diff
@@ -70,7 +70,6 @@ const config = {
       '@': path.join(projectRootDir, 'src'),
       legacyRegistry: path.join(projectRootDir, 'src/legacyRegistry'),
       csv: 'comma-separated-values',
-      EventEmitter: 'eventemitter3',
```

Comment:
woah - big change

Reply:
so i learned that `tsc` and webpack aliases don't play well together at all. After struggling to get `tsc` to recognize that many of our classes extend `EventEmitter` and have `.on()`, `.off()` methods, etc. I decided to just get rid of the alias altogether.

Reply:
in general i think we have a lot of unnecessary aliases